src/types/products.ts

```diff
@@ -115,6 +115,20 @@ interface RootDocsPath {
 	 * content API should pull from. Defaults to `main`.
 	 */
 	mainBranch?: string
+
+	/**
+	 * An optional property to hide the title of this rootDocsPath
+	 * in the sidebar. Used for Terraform routes where sidebar titles
+	 * are present in nav-data.json.
+	 */
+	hideSidebarTitle?: boolean
```

Comment:
Should we clarify that this just _visually_ hides the title? Maybe add it to the name, like `visuallyHideSidebarTitle`?

Reply:
I dig it! ✅ Updated